src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -79,6 +80,7 @@ def load_arguments(self, _):
             c.argument('backup_interval', type=int, help="the frequency(in minutes) with which backups are taken (only for accounts with periodic mode backups)", arg_group='Backup Policy')
             c.argument('backup_retention', type=int, help="the time(in hours) for which each backup is retained (only for accounts with periodic mode backups)", arg_group='Backup Policy')
             c.argument('server_version', arg_type=get_enum_type(ServerVersion), help="Valid only for MongoDB accounts.", is_preview=True)
+            c.argument('default_identity', help="The primary identity to access key vault in CMK related features.", is_preview=True)
```

Comment:
arg_type=get_enum_type(ManagedServiceIdentity)

Reply:
assign_identity should be renamed to ManagedServiceIdentity and its possible values should be None, SystemAssignedIdentity. I user does not provide any value then we will just pass null

Reply:
Have discussed offline. Will keep my old design, and I'll update the description.